test/fixtures/es-modules/package-cjs-named-error/default-and-renamed-import.mjs

```diff
@@ -0,0 +1,4 @@
+import abc, {
+  comeOn as comeOnRenamed,
+  everybody,
```

Comment:
For reasons I don't understand this import statement only errors out on line 3 (`everybody`). I do not understand why it doesn't error out on the renamed import statement. If I swap the order it will error on line 2 (still `everybody`).

Any clues why this is the case?